LoopKitUI/Views/Settings Editors/TherapySettingsViewModel.swift

```diff
@@ -12,11 +12,13 @@ public class TherapySettingsViewModel: ObservableObject {
     private var initialTherapySettings: TherapySettings
     var therapySettings: TherapySettings
     let supportedBasalRates: [Double]?
+    let supportedBolusVolumes: [Double]?
 
-    public init(therapySettings: TherapySettings, supportedBasalRates: [Double]?) {
+    public init(therapySettings: TherapySettings, supportedBasalRates: [Double]?, supportedBolusVolumes: [Double]?) {
```

Comment:
Since the optionality of `supportedBasalRates` is tied to that of `supportedBolusVolumes` (as they both come from PumpManager), you might consider combining the two properties into a single struct that holds them non-optionally, and making the struct reference optional, e.g.

```swift
struct SupportedIncrements {
    var basalRates: [Double]
    var bolusVolumes: [Double]
}

// .. elsewhere:
var supportedIncrements: SupportedIncrements?
```